package.json

```diff
@@ -5,8 +5,11 @@
     "main": "dist/oidc-client.js",
     "scripts": {
         "start": "node sample/server.js",
-        "build": "webpack --config webpack.config.js",
-        "test": "mocha --compilers js:babel-register test/unit/*.spec.js"
+        "build": "webpack -p --config webpack.config.js",
+        "build:classic": "webpack -p --config webpack.config.classic.js",
+        "prebuild": "npm run build:classic",
+        "test": "mocha --compilers js:babel-register test/unit/*.spec.js",
+        "pretest": "npm run build"
```

Comment:
for "pretest" why do we need to run the build? the tests just go after the ES6 modules directly, so they don't need to output from the build.